### DISCUSSION TITLE
Option to not save uninhabited chunks

### DISCUSSION THREAD

**Discussion by zerochill-1** [PaperMC/Paper#10513] [Ideas]
### Is your feature request related to a problem?

Not a problem, but this helps a lot with servers with big world sizes

### Describe the solution you'd like.

A config option in paper-world-defaults.yml that would disable saving chunks to the disk if InhabitedTime is 0
Chunks would still be generated and sent to the player, but if InhabitedTime is 0, the server just doesn't save them to the disk

Another config option that would be useful is do not save chunks that are past the border
If the chunk is past the worldborder, do not save to disk

### Describe alternatives you've considered.

World pruning using MCA selector, which requires downtime

**Comment by Warriorrrr** [comment]
I don't think this is something that needs to be in paper, as it should theoretically be pretty easy to do with a plugin that just listens to the ChunkUnloadEvent, which allows preventing chunks from being saved to disk

**Comment by electronicboy** [comment]
Last I knew, chunks are already saved during chunk generation, cancelling saving here is likely not going to make much of a difference
